pulp_rpm/app/serializers/repository.py

```diff
@@ -119,6 +120,20 @@ class RpmPublicationSerializer(PublicationSerializer):
         ),
     )
 
+    def validate(self, data):
+        """Validate data."""
+        if (
+            data["metadata_checksum_type"] not in settings.ALLOWED_CONTENT_CHECKSUMS
+            or data["package_checksum_type"] not in settings.ALLOWED_CONTENT_CHECKSUMS
+        ):
+            raise serializers.ValidationError(
+                _("Checksum must be one of allowed types: {}").format(
```

Comment:
As a user, I'm not sure what to do when I see this message.
Maybe specifying which configuration specifically causes an issue will help. What do you think?